components/Button/TabButton.tsx

```diff
@@ -19,29 +21,33 @@ export type TabButtonProps = {
 };
 
 const TabButton: React.FC<TabButtonProps> = React.memo(
-	({ title, detail, badge, icon, titleIcon, ...props }) => (
-		<StyledButton noOutline {...props}>
+	({ title, detail, badge, icon, titleIcon, vertical, nofill, ...props }) => (
+		<StyledButton $vertical={vertical} $nofill={nofill} noOutline {...props}>
 			{!!icon && <div>{icon}</div>}
 			<div>
 				<div className="title-container">
 					{titleIcon}
-					<p className="title">{title}</p>
+					<Body className="title">{title}</Body>
 					{!!badge && <div className="badge">{badge}</div>}
 				</div>
 
-				{detail && <p className="detail">{detail}</p>}
+				{detail && (
```

Comment:
Might be good to rename this prop if it's expected to be a number, maybe something like `badgeCount` ?

Reply:
Done in the latest commit.